Azrux/Athenas-front
# Create NavBar component

## Description
Added NavBar component

## Feature
Added heroicons library

## Screenshot
![image](https://github.com/Azrux/Athenas-front/assets/89431511/e6eabd73-2022-41a2-aa52-f3a9718184d3)
![image](https://github.com/Azrux/Athenas-front/assets/89431511/72b08055-a48b-45fd-b2e8-5c7843d7d27d)

## How to test?
* Run `npm run dev`
* Go to: http://localhost:3000/test
* Check the component is working correctly and if you agree with the design
* Check the responsive design
* The NavBar should be hidden when you scroll down the page
